nixnet/_props.py

```diff
@@ -2504,13 +2504,13 @@ def get_cluster_pdu_refs(
     )
 
 
-def get_cluster_pd_us_reqd(
+def get_cluster_pdus_required(
```

Comment:
When I wrote this layer, my intent was for it to reflect the C API and not the python API.  If we decide to expand abbreviations, is it appropriate to do so at this level?

Reply:
Ok, reverted this.